.graphqlconfig

```diff
@@ -0,0 +1,22 @@
+{
+  "projects": {
+    "metaphysics-sdl": {
+      "schemaPath": "data/schema.graphql",
+      "extensions": {
+        "endpoints": {
+          "staging": "https://metaphysics-staging.artsy.net/",
+          "local": "http://localhost:5000"
+        }
+      }
+    },
+    "metaphysics-json": {
+      "schemaPath": "data/schema.json",
+      "extensions": {
+        "endpoints": {
+          "staging": "https://metaphysics-staging.artsy.net/",
+          "local": "http://localhost:5000"
+        }
+      }
+    }
+  }
+}
```

Comment:
yet another GraphQL config file. Given that our graphql extension has been broken for many months it might make sense to remove the other graphql config file.